README.md

```diff
@@ -53,15 +54,15 @@ for users and especially Windows users to create a file compatible with your pro
 
 ## Compatibility Testing
 
-| Version Match | asdf | avn | direnv | fnm | n | nodenv | nodist  | nvm.fish | nvs |
```

Comment:
Formatting was caused by VS Code / Prettier. Very difficult to avoid.

Reply:
I have been caught by that too in projects where I have auto-save enabled, but found a convenient one-off work-around.

View > Command Palette... > File: Save without Formatting